### DISCUSSION TITLE
[RG280v] How to hide the init scripts and statuses while booting in v1.3.1 + SimpleMenu 10.3.1 question

### DISCUSSION THREAD

**Discussion by McNuggetSauce** [eduardofilo/RG350_adam_image#173] [Q&A] [answered]
Hi, I was wondering how to hide the texts that show up while booting the device. I was watching a guide from Retro Game Corps over at youtube and you can see [here](https://youtu.be/5pfYRm34hs0?t=433) that in v1.2 there are no messages (setup permissions, unmount...etc) being shown. It's just the logo and loading bar. Is there a way to achieve this in v1.3.1?

Also, I cannot seem to make the B function to work while holding it. It says in the guide:
> If pressed and held, it acts as a hotkey to be combined with other keys

but it does nothing. Is there a specific section where this only works? I tested with the default SimpleMenu included with the v1.3.1 installation (with OD-beta in the filename) and also replaced with the latest release 10.3.1 but still does not do anything.

Any help is appreciated. Thank you!

**Comment by eduardofilo** [comment, marked as answer]
Hi.

What you see on the screen during boot is generated by the operating system. This operating system was updated between versions 1.2 and 1.3. In the version installed in 1.3 these messages appear. I don't know if there is any adjustment that can be made to avoid them, but they are useful and interesting to me, I don't see that they pose any problem. What is more, what can be done is to show even more information if the `A` key is held down during startup, but not less.

About the use of the `B` key in SimpleMenu as a shortcut, here is the official documentation. It is perhaps better understood than the transcription I made in the image documentation:

https://github.com/fgl82/simplemenu/wiki/3.-Controls

Or maybe you have not understood well that this `B` key is used in conjunction with others, that is, you must press B and without releasing it press other keys to achieve different effects.

I hope it helps.

**Reply by McNuggetSauce** [reply]
Hi Eduardo. I appreciate the response. Thanks for clarifying on both questions.

I guess we can't do much on the boot screen messages. Just thought I ask since it looked cleaner before. However, it's not at all an issue.

On the B button, you are right. I really did misunderstood its function until you mentioned. I thought by holding it, an option is allowed to make it as a hotkey (of my choice). But I did not realize that it only meant that it's actually already a hotkey e.g. B + select for example.

It's all good now. Thank you again and great work!

**Reply by eduardofilo** [reply]
I have changed the description of B key because I do think it could lead to misunderstandings. Thanks.

**Reply by falkenhawk** [reply]
Would it be possible to hardcore in system minimum screen brightness right at the start when system boots, display splashscreen and boot scripts? Asking because when I have last used brightness set and saved to minimum level, it blinds me when I boot the console in the dark, ;) and the brightness is restored to previous only after a while, when scripts are being loaded

**Reply by eduardofilo** [reply]
I haven't looked into it in depth, but I don't think much can be done without recompiling the system, because when the system boots it still won't have mounted the second partition on the card where the brightness level data file is located.
